verification/verification.go

```diff
@@ -38,46 +41,4 @@ var TestCases = []TestCase{
 	TestCase{
 		"GetProfile", TestGetProfile, []string{},
 	},
```

Comment:
optional: Should we put `GetProfile` first because almost all of the other tests depend on it working correctly?